phpunit/bootstrap.php

```diff
@@ -21,6 +21,9 @@
 if ( ! defined( 'LOCAL_WP_ENVIRONMENT_TYPE' ) ) {
 	define( 'LOCAL_WP_ENVIRONMENT_TYPE', 'local' );
 }
+if ( ! defined( 'WP_RUN_CORE_TESTS' ) ) {
```

Comment:
```suggestion
// For why we need this, see https://core.trac.wordpress.org/ticket/57487
if ( ! defined( 'WP_RUN_CORE_TESTS' ) ) {
```

Reply:
Actually, probably not needed if git blame leads back to this PR 😄 

Ignore me

Reply:
I'll add a comment 👍